### ISSUE TITLE
do/while formatting

### ISSUE THREAD

**Issue by frederikhegger** [ros/roslint#43]
In my code I have a do/while loop as follows:

```
do
{
    // do something ....
} while (!msg_received);
```
I get the following error from lint:

```
} should be on a line by itself  [whitespace/braces] [4]
```

But if I move the while statement to a new line, I get the next error:

```
Empty loop bodies should use {} or continue  [whitespace/empty_loop_body] [5]
```

Is there anyway to solve this issue?

**Comment by mikepurvis** [comment]
Would happily accept a PR for this. TBH, I don't really care for the trailing condition while statement; my preferred approach in that case is:

```
while(1)
{

  if (condition) break;
}
```

**Comment by frederikhegger** [comment]
Thanks for the hint. In the meanwhile I did something similar. 

But, If I find some time, I will look into this.

**Comment by alexhenning** [comment]
@frederikhegger I thought I fixed this in https://github.com/ros/roslint/commit/79754bc833209f8aa2e7367da597a76bb6f5923a. Are you running the latest version of roslint?

```
$ cat $(rospack find roslint)/package.xml | grep /version
  <version>0.10.0</version>
```

If you are running the latest, can you post the code that's causing the problem?

**Comment by frederikhegger** [comment]
This might be the problem. Under Ubuntu 12.04 the latest available version (as Debian package) is 0.9.2.

I tried it under Ubuntu 14.04 with version 0.10.0 and everything is fine.

Thanks for clarifying this.
